sig-scheduling/CONTRIBUTING.md

```diff
@@ -72,6 +72,26 @@ to the issue/PR, which helps to preserve the context.
 
 * Always open an issue for a TODO or a follow-up just in case you forget it.
 
+### Technical and style guidelines
+
+The following guidelines apply primarily to kube-scheduler, but some subprojects
+might also adhere to them.
+
+- Follow [effective go](https://go.dev/doc/effective_go) guidelines.
+- Naming:
+  - Length: As a rule-of-thumb, the length of a variable name should be
+    proportional to the size of the scope where it is used and inversely
+    proportional to the number of times that it is used.
+- In tests:
+  - Follow a [DAMP principle](https://stackoverflow.com/a/11837973).
```

Comment:
do you think we should specifically guide users how to write different kinds of tests? basically we would expect the amount of tests to cover a feature is expected to be UT > integration >> e2e. For reproducing a bug, we would be inclined towards writing a UT or integration test over e2e.

Reply:
Added both.